samcli/lib/build/workflow_config.py

```diff
@@ -245,10 +245,7 @@ def _key(c: CONFIG) -> str:
     # A "workflow config" is like a primary key to identify the workflow. So we use the config as a key in the
     # map to identify which workflows can support building within a container.
 
-    unsupported = {
-        _key(DOTNET_CLIPACKAGE_CONFIG): "We do not support building .NET Core Lambda functions within a container. "
-        "Try building without the container. Most .NET Core functions will build "
-        "successfully.",
+    unsupported = { 
```

Comment:
We should ask Mehmet if he thinks this whole method (`supports_build_in_container`) will still be useful in the future, or if we should just remove the whole thing now since all configs support building in container.